docs/src/SUMMARY.md

```diff
@@ -3,5 +3,15 @@
 [Miden](./README.md)
 - [Introduction](./intro.md)
 - [User Documentation](./user_docs.md)
+  - [Miden VM Overview](./user_docs/overview/vm_overview.md)
+  - [Miden Assembly](./user_docs/assembly/miden_assembly.md)
```

Comment:
For the root files - should we use some standard naming convention? e.g. maybe `vm_overview.md` should be `main.md`? And same for `miden_assembly.md`?

Reply:
fixed